samoa-instances/src/main/java/org/apache/samoa/instances/AvroLoader.java

```diff
@@ -254,12 +257,30 @@ protected InstanceInformation getHeader() {
         List<String> attributeLabels = attributeSchema.getEnumSymbols();
         attributes.add(new Attribute(field.name(), attributeLabels));
       }
-      else
+      else if (isNumeric(field))
```

Comment:
I think the idea here is that if we don't know better, we treat the attribute as numeric.
With this change, we ignore attributes we don't know how to treat.
Is my understanding correct?